packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -498,11 +675,16 @@ const ValidationComponent = ( {
 							...edits,
 						} ) )
 					}
+					onValidate={ ( { isValid, isValidating } ) => {
+						setIsFormValid( isValid );
+						setIsFormBusy( isValidating );
+					} }
 				/>
 				<Button
 					__next40pxDefaultSize
 					accessibleWhenDisabled
-					disabled={ ! canSave }
+					disabled={ isFormBusy || ! isFormValid }
+					isBusy={ isFormBusy }
```

Comment:
Can a button be busy and disabled, @mirka? Note this weird interaction:

- Button is disabled because form is invalid.
- Some interaction triggers an async validation: button becomes busy, and changes appearance to "active".
- Async validation resolves and button goes back to invalid.


https://github.com/user-attachments/assets/ac46430d-3e59-4d65-9486-fa187fb98a7b